runtime/serde/serde-xml/common/src/aws/smithy/kotlin/runtime/serde/xml/XmlDeserializer.kt

```diff
@@ -226,10 +226,13 @@ private class XmlStructDeserializer(
 
     private val reader: XmlStreamReader = if (unwrapped) reader else reader.subTreeReader(XmlStreamReader.SubtreeStartDepth.CHILD)
 
+    companion object {
```

Comment:
You don't need a companion object for this. Constants are compile time replacements. This declares an empty companion object essentially.